src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+public class Circle extends Figure {
+    private static final DecimalFormat decimalFormat = new DecimalFormat("#.##");
```

Comment:
java code convention violations
```suggestion
    private static final DecimalFormat decimalFormat = new DecimalFormat("#.##");
```